addons/contour/1.0.1/install.sh

```diff
@@ -16,6 +16,11 @@ function contour() {
 
     cp "$src/patches/service-patch.yaml" "$dst/"
 
+    if [ -z "$CONTOUR_TLS_MINIMUM_PROTOCOL_VERSION" ]; then
+        CONTOUR_TLS_MINIMUM_PROTOCOL_VERSION="1.2"
```

Comment:
this will default the version to 1.2. im not sure if this is desired or not

Reply:
I think it is, 1.1- is being deprecated everywhere